interface/src/ui/DialogsManager.cpp

```diff
@@ -130,15 +131,19 @@ void DialogsManager::hideLoginDialog() {
 
 
 void DialogsManager::showDomainLoginDialog() {
+    const QJsonObject& settingsObject = DependencyManager::get<NodeList>()->getDomainHandler().getSettingsObject();
+    static const QString WP_OAUTH2_SERVER_URL = "authentication_oauth2_url_base";
+    
+    if (!settingsObject.contains(WP_OAUTH2_SERVER_URL)) {
+        qDebug() << "Cannot log in to domain because an OAuth2 authorization was required but no authorization server was given.";
```

Comment:
```suggestion
        qDebug() << "Cannot log in to domain because an OAuth2 authorization was required but no authorization server was specified.";
```